src/resources/views/crud/columns/array_count.blade.php

```diff
@@ -1,24 +1,24 @@
 {{-- enumerate the values in an array  --}}
 @php
     $array = data_get($entry, $column['name']);
-    $suffix = isset($column['suffix']) ? $column['suffix'] : 'items';
+
+    $column['escaped'] = $column['escaped'] ?? false;
+    $column['prefix'] = $column['prefix'] ?? '';
+    $column['suffix'] = $column['suffix'] ?? 'items';
+    $column['text'] = '-';
```

Comment:
For future us, we should remove this hardcoded value `-` (dash)!

Reply:
I agree 🙌
That's why I moved all this default values (the dashes) to the top 😅
Maybe this should be set on configs 🤷‍♂️

We should do it when we refactor fields/columns on v4.2.
There are some things to improve on this matter.